src/AspNet.Security.OAuth.WorldID/WorldIDAuthenticationConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.WorldID;
+
+/// <summary>
+/// Contains constants specific to the <see cref="WorldIDAuthenticationHandler"/>.
+/// </summary>
+public static class WorldIDAuthenticationConstants
```

Comment:
As these appear to be in beta and aren't used anywhere but the place we should remove their usage, we should also remove this entire class.